app/src/test/scala/org/alephium/explorer/persistence/DatabaseFixture.scala

```diff
@@ -40,10 +40,10 @@ object DatabaseFixture {
   def createDatabaseConfig(): DatabaseConfig[PostgresProfile] =
     DatabaseConfig.forConfig[PostgresProfile]("db", config)
 
-  def dropCreateTables(databaseConfig: DatabaseConfig[PostgresProfile]) = {
-    val dbInitializer: DBInitializer = new DBInitializer(databaseConfig)(ExecutionContext.global)
+  def dropCreateTables()(implicit databaseConfig: DatabaseConfig[PostgresProfile]) = {
+    implicit val ec: ExecutionContextExecutor = ExecutionContext.global
```

Comment:
why can't it just be an `ExecutionContext` ? it's just "cleaner" to have less import :stuck_out_tongue_closed_eyes: